.gitignore

```diff
@@ -15,7 +15,9 @@
 # User-specific stuff:
 .idea/workspace.xml
 .idea/tasks.xml
+.idea/findbugs-idea.xml
```

Comment:
This won't work, because of https://github.com/OfficeDev/ews-java-api/issues/235#issuecomment-90041202.